### PR TITLE
Generate Fedora spec file too

### PR DESCRIPTION
**Pull request by superm1**

### CODE REVIEW

File: contrib/ci/dependencies.xml [hunk 1]
```diff
@@ -5,8 +5,17 @@
...
+  <dependency type="runtime" id="bubblewrap">
+    <distro id="fedora">
+      <control />
+      <package>bubblewrap</package>
+    </distro>
+  </dependency>
   <dependency type="build" id="adobe-source-han-sans-cn-fonts">
     <distro id="fedora">
+      <control>
+        <inclusive>have_uefi</inclusive>
```
Comment (hughsie) [inline contrib/ci/dependencies.xml]: I'm a bit confused why we need this. If the .spec.in is just for the CI, and we're only building with UEFI then we can just chose some sane defaults for travis and drop the other bits.

Comment (superm1) [reply contrib/ci/dependencies.xml]: Well I would think the results that you generate from spec.in would also be the "basis" of what you would use for Fedora and rhel and such. If that's a wrong assumption then yeah this part isn't needed.

Comment (hughsie) [reply contrib/ci/dependencies.xml]: I was prepared to break the link between "what ships upstream in spec.in" and "what we ship in fedora" -- as we don't actually write the .spec.in->.spec to disk anywhere when building. I think trying to handle the `inclusive` stuff kinda breaks down when you're handling the meson configure line.

Comment (superm1) [reply contrib/ci/dependencies.xml]: Writing .spec is just running
```OS=fedora contrib/ci/generate_fedora_spec.py contrib/fwupd.spec.in fwupd.spec```
Which I think can easily be part of your release process for RPMs, or even better have the create ```ninja dist``` command also generate it?

The meson configure line is going to have to be hardcoded for all packaging, I don't think there is a way around that.  I think just sticking to dependency generation is a win though.  If a particular dependency needs to be guarded in an ```<inclusive>``` block ***and*** ```meson``` configure line, adding the meson configure line shouldn't be the end of the world as part of the commit.

Comment (hughsie) [reply contrib/ci/dependencies.xml]: I think if ninja generated a build/contribute/fwupd.spec that would be usable from my point of view.

**Pull request closed without merging**